Containers/Landmark/Landmark.js

```diff
@@ -34,9 +35,12 @@ export class Landmark extends Component {
                     this.state.collectedLandmarks.map((landmark => {
                         return (
                             <View key={landmark.id} style={styles.landmarkContainer}>
-                                <Text>Landmark name: {landmark.name}</Text>
-                                <Text>Point value: {landmark.pointVal}</Text>
-                                <Text>Date visited: {landmark.visited}</Text>
+                                <Image source={require('../../assets/statueofliberty.jpg')} style={{ width: 300, height: 200, bottom: 15 }}/>
+                                <Text style={{ fontWeight: 'bold' }}>{landmark.name.toUpperCase()} ({landmark.visited})</Text>
+                                <View style={{ display: 'flex' }}>
+                                    <Icon color="#f44336" name="diamond" type="font-awesome" size={20} left={-140} top={3}/>
+                                    <Text style={{ top: -15, left: 30}}>{landmark.pointVal} gems</Text>
```

Comment:
I think we might have to try to implement flex instead of using pixel positions. It looks good on my phone, but when I look at it in the IOS simulator with iPhone xR it doesn't look right. Since it looks good on our phones I think we should move ahead as is to complete our MWP and then start to do some progressive enhancement styling wise if we have time.